boards/st/stm32u5g9j_dk2/stm32u5g9j_dk2.dts

```diff
@@ -311,6 +327,42 @@ zephyr_udc0: &usbotg_hs {
 	};
 };
 
+&xspi1 {
+	clocks = <&rcc STM32_CLOCK(AHB2_2, 12U)>,
+		<&rcc STM32_SRC_PLL2_Q HSPI_SEL(2)>;
+
+	pinctrl-0 = <&hspi1_dqs0_pi2 &hspi1_ncs_ph9
+		     &hspi1_io0_ph10 &hspi1_io1_ph11
+		     &hspi1_io2_ph12 &hspi1_io3_ph13
+		     &hspi1_io4_ph14 &hspi1_io5_ph15
+		     &hspi1_io6_pi0 &hspi1_io7_pi1
+		     &hspi1_clk_pi3>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	mx66lm1g45: xspi-nor-flash@0 {
+		compatible = "st,stm32-xspi-nor";
+		reg = <0>;
+		size = <DT_SIZE_M(128)>; /* 1 Gbits */
+		ospi-max-frequency = <DT_FREQ_M(133)>;
+		spi-bus-width = <XSPI_OCTO_MODE>;
+		data-rate = <XSPI_DTR_TRANSFER>;
+		four-byte-opcodes;
+		status = "okay";
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			extflash_partition: partition@0 {
+				label = "extflash";
```

Comment:
```suggestion
				label = "storage";
```
This will help testing the partition in related flash tests/samples which expect a storage partition.